extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -86,9 +84,9 @@ export class WindowPortEmulator {
   addEventListener(eventType, handler) {
     this.win.addEventListener('message', e => {
       if (
-        e.origin == this.origin_ &&
-        e.source == this.target_ &&
-        getData(e)['app'] == APP
+        e.origin === this.origin_ &&
```

Comment:
The `===` seems to be overused here a bit. Why?

Reply:
I figured it's good practice to prefer it over `==` even when the type is known, but happy to revert these bits if that's not a good idea.

Reply:
No, I think we generally prefer strict typing and just `==`. This library is exported out so some proofing is not a bad idea, but when we are certain of types it's still distracting.

Reply:
Got it. Reverted to `==` where the type is known.